src/test/components/__snapshots__/Root.test.js.snap

```diff
@@ -2,6 +2,115 @@
 
 exports[`app/ProfileViewWhenReady does not try to retrieve a profile when moving from from-addon to public 1`] = `<profile-viewer />`;
 
+exports[`app/ProfileViewWhenReady renders a compare home when navigating to /compare/, then loads when changing profiles 1`] = `
+<main
+  class="compareHome"
+>
+  <h1
+    class="compareHomeTitle"
+  >
+    Enter the profile URLs that you’d like to compare
+  </h1>
+  <p>
+    The tool will extract the data from the selected track and range for each profile, and put them both on the same view to make them easy to compare.
+  </p>
+  <form
+    class="compareHomeForm"
+  >
+    <label
+      class="compareHomeFormLabel"
+      for="compareHomeProfile1"
+    >
+      Profile 1:
+    </label>
+    <input
+      class="photon-input"
+      id="compareHomeProfile1"
+      name="profile1"
+      placeholder="http://"
+      required=""
+      type="url"
+      value=""
+    />
+    <label
+      class="compareHomeFormLabel"
+      for="compareHomeProfile2"
+    >
+      Profile 2:
+    </label>
+    <input
+      class="photon-input"
+      id="compareHomeProfile2"
+      name="profile2"
+      placeholder="http://"
+      required=""
+      type="url"
+      value=""
+    />
+    <input
+      class="compareHomeSubmitButton"
+      type="submit"
+      value="Retrieve profiles"
+    />
+  </form>
+</main>
+`;
+
+exports[`app/ProfileViewWhenReady renders a compare home when navigating to /compare/, then loads when changing profiles 2`] = `
+<div
```

Comment:
Yeah, I feel like we don't need a snapshot here to confirm that the loader shows.